src/models/curriculum/problem.ts

```diff
@@ -30,6 +30,7 @@ const ModernProblemModel = types
      * with all of the other sections in this problem, or this problem's unit
      */
     sectionsFromSnapshot: types.frozen<SectionModelSnapshot[]>(),
+    exemplarPaths: types.array(types.string),
```

Comment:
I'd recommend just calling this property `exemplars` and removing the changes to the preProcessor snapshot below. Since we are not storing this in a volatile property or providing them as a view, there isn't a need to do the fancying mapping like sectionsFromSnapshot.